python/ray/data/impl/compute.py

```diff
@@ -234,9 +234,12 @@ def cache_wrapper(
     """
     if isinstance(fn, CallableClass):
 
-        if compute is None:
+        if compute is None or (
+            compute != "actors" and not isinstance(compute, ActorPoolStrategy)
+        ):
```

Comment:
Can we specifically blacklist `tasks` instead of whitelisting actors? This allows us to put in other compute strategies in the future.

Reply:
Good point!